### PR TITLE
Changes to devel

### PR DESCRIPTION
**Pull request by delarosatrevin**
### Main changes:
* removed the 'replace' command related to the environment variables
* Added version 1.8.2 for both GPU and CPU environments
* removed 'boxSizeEstimated' stored property, it can be inferred from the boxSize value
* Refactored a bit coordinates parsing:
   * Avoid creating one reader for each micrograph
   * Avoid instantiating a Float object every time
   * Make more clear modification of the outputCoords set outside the function

### CODE REVIEW

**Review by pconesa — APPROVED** [review]
All looks correct to me. Test of the picking with both versions and its installation worked in my machine

File: sphire/__init__.py [hunk 1]
```diff
@@ -158,10 +155,10 @@ def addCryoloPackage(cls, env, version, default=False, useCpu=False,
         installationCmd = cls.getCondaActivationCmd()
 
         # Creating the environment
-        if version == V1_8_0:
+        if version in [V1_8_0, V1_8_1]:
```
[the remark this answers is not in the document]
Comment (delarosatrevin) [reply sphire/__init__.py]: Pablo, I updated to 1.8.2 today, since Thorsten fixed a couple of things...so we might use that one instead of 1.8.1

**Comment by delarosatrevin** [comment]
@pconesa there is a test failing related to estimated boxsize and the scale factor that you added. My guess is that might the estimated size changed with 1.8.x version and then you expected result changed. Can you take a look at that?

File: sphire/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (delarosatrevin) [reply sphire/__init__.py]: @pconesa I removed version 1.8.0 as you suggested, I agree that it does not make sense to keep it if we have 1.8.2

**Pull request merged**